.travis.yml

```diff
@@ -12,7 +12,10 @@ git:
   depth: 1
 
 env:
-  - RISCV=$TRAVIS_BUILD_DIR/riscv PATH=$PATH:$RISCV/bin KEYSTONE_SDK_DIR=$TRAVIS_BUILD_DIR/keystone-sdk
+  - RISCV=$TRAVIS_BUILD_DIR/riscv
+    PATH=$PATH:$RISCV/bin
+    KEYSTONE_SDK_DIR=$TRAVIS_BUILD_DIR/keystone-sdk
```

Comment:
KEYSTONE_SDK_DIR should be the install directory if the SDK now.
See README.md of `https://github.com/keystone-enclave/keystone-sdk` (master)
so replace this with `KEYSTONE_SDK_DIR=$TRAVIS_BUILD_DIR/keystone-sdk/build`